Source/Urho3D/Physics/PhysicsWorld.cpp

```diff
@@ -88,6 +89,26 @@ static bool CustomMaterialCombinerCallback(btManifoldPoint& cp, const btCollisio
     return true;
 }
 
+void RemoveCachedGeometryImpl(CollisionGeometryDataCache& triMeshCache_, Model* model)
+{
+    for (auto i = triMeshCache_.Begin(); i != triMeshCache_.End();)

```

Comment:
I'll fix it.

Reply:
BTW, I cannot. Iterator is needed here.

Reply:
I believe I saw one of the iterator-based for loop being replaced correctly by Clang modernizer in the development branch.

Reply:
> I believe I saw one of the iterator-based for loop being replaced correctly by Clang modernizer in the development branch.

I've seen it do that as well, unsure why it did not do it in this case.

Reply:
@weitjong @OvermindDL1 Feel free to fix it if you are able to. Unfortuantelly, I am not.

Reply:
I wonder if it is because it is not fulfilling the std-standard iterator interface (`begin`/`end` instead of the really odd `Begin`/`End`?).  Probably special cased to follow the standards terminology and naming.

Reply:
@OvermindDL1 Nope, it's just because the code couldn't be converted into range-based form.

Reply:
> I've seen it do that as well, unsure why it did not do it in this case.

@OvermindDL1 It is simply because my clang-tidy work is on a separate branch which does not contain this commit yet.